modules/ols-byok-task.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="byok-task_{context}"]
+= Adding custom knowledge to the LLM
+
+Customize the information available to the large language model (LLM) by providing access to a container image that resides in a remote image registry. The examples in this procedure use `quay.io` as the remote container image registry, and the path for the custom image is `quay.io/<username>/my-byok-image:latest`.
+
+[NOTE]
+====
+Adding custom knowledge to the LLM is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR file.
+
+* You have configured the large language model (LLM) provider.
+
+* You have installed the {ols-long} Operator.
+
+* The custom information you want to add resides in a Markdown file; no other file format is supported.
```

Comment:
a Markdown file -> a collection of Markdown files

Reply:
done